homepage/homepage/app/(docs)/docs/[framework]/[...slug]/authentication/overview.mdx

```diff
@@ -0,0 +1,404 @@
+export const metadata = { title: "Authentication methods" };
+
+import { CodeGroup, ContentByFramework } from "@/components/forMdx";
+
+# Authentication in Jazz
+
+Jazz authentication is based on cryptographic keys that are stored locally on the user device.
+
+When a user loads a Jazz application for the first time, we generate those keys and store them locally.
+
+Without any additional steps the user can use Jazz normally, but they would be limited to use on only one device.
+
+For an account to use multiple devices, and back up their credentials, you can set up an authentication method using one of the Jazz providers.
+
+<ContentByFramework framework={["react", "vue", "svelte"]}>
+## Authentication with passkeys
+
+Passkey authentication is fully local-first and the most secure of the auth methods that Jazz provides.
```

Comment:
```suggestion
Passkey authentication is fully local-first and the most secure of the auth methods that Jazz provides (because keys are manageed by the device/operating system itself).
```